examples/browser/package.json

```diff
@@ -4,69 +4,57 @@
   "version": "0.0.1-alpha.1",
   "license": "Apache-2.0",
   "dependencies": {
-    "monaco-css": "^1.3.1",
-    "monaco-editor": "^0.8.3",
-    "monaco-editor-core": "^0.8.2",
-    "monaco-html": "^1.2.1",
-    "monaco-json": "^1.2.2",
-    "monaco-languages": "^0.7.0",
-    "monaco-typescript": "^2.1.2",
-    "reflect-metadata": "^0.1.10",
-    "theia-core": "latest",
-    "yargs": "^8.0.1"
+    "theia-core": "latest"
   },
   "localDependencies": {
     "theia-core": "../.."
   },
   "scripts": {
     "bootstrap": "npm install ../../config/local-dependency-manager && npm run localinstall",
-    "localinstall": "ldm install --dev --original-sources",
     "clean": "rimraf lib",
     "prepare": "npm run clean && npm run build",
-    "build": "npm run build:localdeps && npm run build:backend && npm run build:frontend",
-    "build:backend": "tsc && npm run lint",
-    "build:frontend": "webpack && cp src/frontend/index.html lib/frontend",
-    "build:localdeps": "ldm run build && ldm sync --original-sources",
-    "lint": "tslint -c ../../config/tslint/tslint.json --project ./tsconfig.json",
+    "cold:start": "npm run clean && npm run build && npm start",
+    "build": "npm run build:localdeps && npm run build:frontend",
+    "build:frontend": "webpack --target web && cp src-gen/frontend/index.html lib",
+    "watch": "concurrently --names watch-deps,watch-frontend --prefix \"[{name}]\" \"npm run watch:localdeps\" \"npm run watch:frontend\"",
+    "watch:frontend": "npm run build:frontend && webpack --target web --watch",
     "start": "concurrently --names backend,webpack-server --prefix \"[{name}]\" \"npm run start:backend\" \"npm run start:frontend\"",
-    "start:backend": "node ./lib/backend/main.js | bunyan",
-    "start:backend:debug": "node ./lib/backend/main.js --loglevel=debug | bunyan",
+    "start:backend": "node ./src-gen/backend/main.js | bunyan",
+    "start:backend:debug": "node ./src-gen/backend/main.js --loglevel=debug | bunyan",
     "start:frontend": "webpack-dev-server --open",
-    "cold:start": "npm run clean && npm run build && npm start",
-    "watch": "concurrently --names watch-deps,watch-backend,watch-frontend --prefix \"[{name}]\" \"npm run watch:localdeps\" \"npm run watch:backend\"  \"npm run watch:frontend\"",
-    "watch:backend": "tsc --watch",
-    "watch:frontend": "npm run build:frontend && webpack --watch",
+    "localinstall": "ldm install --dev --original-sources",
+    "build:localdeps": "ldm run build && ldm sync --original-sources",
     "watch:localdeps": "ldm watch --sync --run=watch --original-sources",
     "test:ui": "wdio wdio.conf.js"
   },
   "devDependencies": {
-    "@types/chai": "^3.5.1",
-    "@types/webdriverio": "^4.7.0",
-    "@types/yargs": "^6.6.0",
-    "bunyan": "^1.8.10",
-    "chai": "^3.5.0",
+    "rimraf": "^2.6.1",
```

Comment:
This file is auto-generated, one should change `theia.package.json` and run `yo theia:browser --force`.